app/src/main/java/com/globant/example/mentorapp/mvp/base/BasePresenter.java

```diff
@@ -0,0 +1,25 @@
+package com.globant.example.mentorapp.mvp.base;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+
+import com.squareup.otto.Bus;
+
+/**
+ * Base class to define all methods common to any presenter.
+ * Created by oscar.amezquita on 16/06/2017.
+ */
+
+public class BasePresenter {
```

Comment:
Add attach and detach methods to this class. So, any Presenter will extend them and you avoid duplicated code.